### ISSUE TITLE
Need some facets on MVP2 results

### ISSUE THREAD

**Issue by sstemann** [NCATSTranslator/Feedback#537]
Genes don't have any facets - is there a list somewhere of what was discussed?

https://ui.ci.transltr.io/demo/results?l=Eletriptan&i=PUBCHEM.COMPOUND:77993&t=3&q=d1c6af0a-8f60-4f24-87fe-a5268c338cea

![image](https://github.com/NCATSTranslator/Feedback/assets/38321826/62e2a503-ce61-4e8e-ada7-cf2291a02c51)

**Comment by Genomewide** [comment]
Do you mean the chebi roles? Genes don't have them if so.  The other facets like FDA don't apply here either.

**Comment by sstemann** [comment]
For MVP2 0 Find Genes regulated by a particular chemical, a starting point to investigate would be:

tissues or cells
gene regulatory networks
chromosome
Genome Browser has categories

Not sure what's currently available in Annotator or mygene.info @newgene

**Comment by Genomewide** [comment]
Did a review of mygene.info and found GO and pathways.  I don't think tissue is available here. @newgene Is that right? 
If not, who is the GTEX contact?

**Comment by newgene** [comment]
@Genomewide you are right, we don't have tissue info at mygene.info. We do have the genomic location, and this `type_of_gene` field might be useful too:

https://mygene.info/v3/query?species=human&facets=type_of_gene&facet_size=20

**Comment by newgene** [comment]
How about these GO categories?

https://mygene.info/v3/gene/5871?fields=go

**Comment by Genomewide** [comment]
yeah those go categories would work.  We would probably have to create our own go slim or filter for high level terms
